<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ges_medicament</class>
 <widget class="QMainWindow" name="ges_medicament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1863</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1931</width>
      <height>931</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>311</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 55, 98);</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_IDM">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Libelle">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prix">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Quantite">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>IDM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Libellé</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Quantité</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_A">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_U">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_D">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Export">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>Export PDF Medicament</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Fournisseur">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>200</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(166, 227, 255);</string>
       </property>
       <property name="text">
        <string>FOURNISSEUR</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>511</width>
        <height>405</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 55, 98);
color: rgb(255, 255, 255);
QHeaderView::section {
    background-color: rgb(0, 128, 129);
    color: white; 
    font-weight: bold;
    border: 1px solid black; 
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_T">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Recher">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Recher">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
      </property>
      <property name="text">
       <string>Recherch</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>80</y>
        <width>530</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ShowRevenue">
      <property name="geometry">
       <rect>
        <x>1330</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
      </property>
      <property name="text">
       <string>Revenues</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IDM_Check">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>560</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CheckStock">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>560</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 128, 129);
color: rgb(255, 255, 255);
border-radius: 12;
font-weight: bold;
border: none;</string>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>231</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 55, 98);</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_ToEmployes">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 244);
border-radius: 13;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>EMPLOYE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Clientm">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 244);
border-radius: 13;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>CLIENT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Fournisseurm">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 244);
border-radius: 13;
font-weight: bold;
border: none;</string>
       </property>
       <property name="text">
        <string>FOURNISSEUR</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1670</x>
        <y>510</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1863</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
